Products/Products-Root/Utils/Navigation/AuthenticationCoordinator.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  AuthenticationCoordinator.swift
+//  Products
+//
+//  Created by Dan Pop on 11/08/2020.
+//  Copyright © 2020 Archlime. All rights reserved.
+//
+
+import UIKit
+
+protocol AuthenticationCoordinatorDelegate: class {
+    func authenticationCoordinatorDidLogin()
+}
+
+class AuthenticationCoordinator: Coordinator {
+
+    private let navigationController: UINavigationController
+    private weak var delegate: AuthenticationCoordinatorDelegate?
+
+    // MARK: - Initializer
+    init(navigationController: UINavigationController, delegate: AuthenticationCoordinatorDelegate) {
+        self.navigationController = navigationController
+        self.delegate = delegate
+    }
+
+    // MARK: - Navigation
+    func start() {
+        navigationController.setNavigationBarHidden(true, animated: false)
+        pushSplash()
+    }
+
+    // MARK: - Private
+    private func pushSplash() {
```

Comment:
rename to showSplash. Naming conventionurile sunt asa: 

- daca se face "setViewControllers" numim metoda "showX"
- daca se face "pushViewController" numim metoda "pushX"
- daca se face "presentViewController" numim metoda "presentX"